web-test-runner.utils.js

```diff
@@ -277,32 +278,3 @@ export const configuredVisualRegressionPlugin = () =>
             );
         },
     });
-
-export function watchSWC() {
```

Comment:
Do you know what this was doing or why its safe to remove this now?

Reply:
Yes this was forcing rebuild of the assets when updates were made to the source files. The current system handles this automatically and doesn't need us to manually kick this off any more. We can validate this by making changes to an existing asset and seeing it live refresh on the page. I'll add a test case for that when I start writing up validation steps.

Reply:
worked for me!